project.Rproj

```diff
@@ -18,5 +18,4 @@ StripTrailingWhitespace: Yes
 BuildType: Package
 PackageUseDevtools: Yes
 PackageInstallArgs: --no-multiarch --with-keep.source
-PackageCheckArgs: --as-cran
```

Comment:
Why is this line removed?

Reply:
> Why is this line removed?

I'm not sure, my build settings seem right, and it passes all the R CMD check 1 NOTE:

<img width="597" alt="Screen Shot 2021-02-23 at 8 48 14 AM" src="https://user-images.githubusercontent.com/31193655/108860568-f0886b00-75b3-11eb-9484-4272eeedbafb.png">

```
> checking for portable file names ... NOTE
  Found the following non-portable file path:
    hySpc.chondro/vignettes/hySpc-chondro_cache/html/chondro-import_159d9d14e642eb48cb4935f7e3a62bd5.RData
```